devicetypes/smartthings/nyce-open-closed-sensor.src/nyce-open-closed-sensor.groovy

```diff
@@ -299,51 +259,18 @@ def configure() {
 	// Device-Watch allows 2 check-in misses from device
 	sendEvent(name: "checkInterval", value: 60 * 12, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID, offlinePingable: "1"])
 
-	String zigbeeEui = swapEndianHex(device.hub.zigbeeEui)
-
-	def enrollCmds = [
-			// Writes CIE attribute on end device to direct reports to the hub's EUID
-			"zcl global write 0x500 0x10 0xf0 {${zigbeeEui}}", "delay 200",
-			"send 0x${device.deviceNetworkId} 1 1", "delay 500",
-	]
-
-	log.debug "configure: Write IAS CIE"
-	// battery minReportTime 30 seconds, maxReportTime 5 min. Reporting interval if no activity
-	return enrollCmds + zigbee.batteryConfig(30, 300) + refresh() // send refresh cmds as part of config
-}
-
-def enrollResponse() {
-	[
-			// Enrolling device into the IAS Zone
-			"raw 0x500 {01 23 00 00 00}", "delay 200",
-			"send 0x${device.deviceNetworkId} 1 1"
-	]
+	if(device.getDataValue("manufacturer") == "sengled") {
+		return zigbee.readAttribute(zigbee.IAS_ZONE_CLUSTER, zigbee.ATTRIBUTE_IAS_ZONE_STATUS) + zigbee.readAttribute(zigbee.POWER_CONFIGURATION_CLUSTER, 0x0020) + zigbee.batteryConfig(30, 300) + zigbee.enrollResponse()
+	} else {
+		// battery minReportTime 30 seconds, maxReportTime 5 min. Reporting interval if no activity
+		return zigbee.iasZoneConfig() + zigbee.batteryConfig(30, 300) + refresh() // send refresh cmds as part of config
+	}
 }
 
 private hex(value) {
```

Comment:
I think this is unused now and can be removed

Reply:
sure, removed